libraries/integration/Microsoft.Bot.Builder.Integration.Runtime/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -34,10 +37,14 @@ public static class ServiceCollectionExtensions
         /// THIS IS TEMPORARY TEST CODE. Adds bot runtime-related services to the application's service collection.
         /// </summary>
         /// <param name="services">The application's collection of registered services.</param>
-        public static void AddAdaptiveRuntime(this IServiceCollection services)
+        /// <param name="configuration">IConfiguration.</param>
+        public static void AddAdaptiveRuntime(this IServiceCollection services, IConfiguration configuration)
         {
             _ = services ?? throw new ArgumentNullException(nameof(services));
 
+            var component = new Dialogs.DialogsBotComponent();
```

Comment:
We can leave this for testing now but these 2 lines need to go when we wrap up this workstream.

Reply:
yes next PR!